src/app/components/canvas/canvas.component.ts

```diff
@@ -211,25 +222,21 @@ export class CanvasComponent implements OnChanges, OnInit, OnDestroy {
     }
 
     private handlePassedElement(draggable: Draggable) {
-        const passedElement = this._stateHandler.getActiveNeighbourElement()
-            ?.transition as HTMLElement;
-        const movingElement = draggable.transition;
+        const passedElement = this._stateHandler.getActiveNeighbourElement();
+        const movingElement = draggable.event;
         if (passedElement === undefined) {
             return;
         }
-        MoveElementsService.switchElements(
-            draggable,
-            this._draggingService.createDraggable(passedElement) as Draggable
-        );
-        if (this.persistCoordinates) {
```

Comment:
Ich habe das so umgebaut, dass die Koordinaten bei der Merge-View nur nicht in die Textarea geschrieben werden. Der Text im Run wird jedoch aktualisiert, sodass das beim Download auch entsprechend berücksichtigt wird.